packages/thumbnail/src/spectrum-config.v2.js

```diff
@@ -0,0 +1,53 @@
+// @ts-check
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { converterFor } from '../../../tasks/process-spectrum-utils.js';
+
+const converter = converterFor('spectrum-Thumbnail');
+
+/**
+ * @type { import('../../../tasks/spectrum-css-converter').SpectrumCSSConverter }
+ */
+export default {
+    conversions: [
+        {
+            inPackage: '@spectrum-css/thumbnail',
+            outPackage: 'thumbnail',
+            fileName: 'thumbnail',
+            components: [
+                converter.classToHost(),
+                converter.classToAttribute('is-selected', 'selected'),
+                converter.classToAttribute('is-focused', 'focused'),
+                converter.classToAttribute(
+                    'spectrum-Thumbnail--cover',
```

Comment:
This can currently be `converter.classToAttribute('spectrum-Thumbnail--cover')`. Did not get to some of your more nuanced stuff you mentioned in our chat before. Should work with the `background` class below, too.

Reply:
nice, I think it's plenty already - just need to learn when the "key" you construct it with can be omitted, and when it can't be

Reply:
If the source is `${key}-${value}` or `${key}--${value}` and you want it to be `value`, you can leverage this.